apps/studio/src/features/editing-experience/components/CreateCollectionModal/CreateCollectionModal.tsx

```diff
@@ -0,0 +1,185 @@
+    // This allows the syncing to happen only when the page title is not dirty
+    // Dirty means user has changed the value AND the value is not the same as the default value of "".
+    // Once the value has been cleared, dirty state will reset.
+    if (!permalinkFieldState.isDirty) {
+      setValue("permalink", generateResourceUrl(collectionTitle), {
+        shouldValidate: !!collectionTitle,
+      })
+    }
+  }, [getFieldState, setValue, collectionTitle])
+
+  return (
+    <ModalContent key={String(isOpen)}>
+      <form onSubmit={onSubmit}>
+        <ModalHeader>Create a new collection</ModalHeader>
+        <ModalCloseButton size="sm" />
+        <ModalBody>
+          <VStack alignItems="flex-start" spacing="1.5rem">
+            <FormControl isInvalid={!!errors.collectionTitle}>
+              <FormLabel color="base.content.strong">
+                Collection name
+                <FormHelperText color="base.content.default">
+                  This will be the title of the index page of your collection.
+                </FormHelperText>
+              </FormLabel>
+
+              <Input
+                placeholder="This is a title for your new collection"
+                {...register("collectionTitle")}
+              />
+              {errors.collectionTitle?.message ? (
+                <FormErrorMessage>
+                  {errors.collectionTitle.message}
+                </FormErrorMessage>
+              ) : (
+                <FormHelperText mt="0.5rem" color="base.content.medium">
+                  {MAX_FOLDER_TITLE_LENGTH - collectionTitle.length} characters
+                  left
+                </FormHelperText>
+              )}
+            </FormControl>
+            <FormControl isInvalid={!!errors.permalink}>
+              <FormLabel color="base.content.strong">
+                Collection URL
+                <FormHelperText color="base.content.default">
+                  This will be applied to every child under this collection.
+                </FormHelperText>
+              </FormLabel>
+              <Input
+                placeholder="This is a url for your new page"
+                {...register("permalink")}
+              />
+              {errors.permalink?.message && (
+                <FormErrorMessage>{errors.permalink.message}</FormErrorMessage>
+              )}
+
+              <Box
+                mt="0.5rem"
+                py="0.5rem"
+                px="0.75rem"
+                bg="interaction.support.disabled"
+              >
+                <Icon mr="0.5rem" as={BiLink} />
+                {permalink}
+              </Box>
+
+              <FormHelperText mt="0.5rem" color="base.content.medium">
+                {MAX_FOLDER_PERMALINK_LENGTH - permalink.length} characters left
+              </FormHelperText>
+            </FormControl>
+          </VStack>
+        </ModalBody>
+
+        <ModalFooter>
+          <Button mr={3} onClick={onClose} variant="clear">
```

Comment:
nit: we should use `rem` instead of a number

Reply:
what's the quickmath again

Reply:
idk LOL you inspect element and tell me

Reply:
shit i thought you had the conversion tables memorised :sadge: